### PR TITLE
Bug: absent item fixed

### LINKED ISSUE #140: Products out of stock in menu
Products that are out of stock are still being shown in the menu, allowing users to go into the product and add it to the cart.
Products that are out of stock should be shown the same way the products offered only on Fridays look on a Monday, with the difference of adding a text similar to "(Em falta)" like the text for products offered on specific days "(Sextas)".
image
image

### PR DESCRIPTION
**Pull request by ericbraz**
Resolves #140

### CODE REVIEW

File: components/MenuList/ListingItems/ListingItems.tsx [hunk 1]
```diff
@@ -20,15 +20,17 @@ export default function Listing({ item, isPromotionActive }: Props) {
       })
    }, [])
 
+   const activation = activeItem&&item.isAvailable
```
Comment (brunorplima) [inline components/MenuList/ListingItems/ListingItems.tsx:23]: Aren't you getting an error on this line? There is no space

Comment (ericbraz) [reply components/MenuList/ListingItems/ListingItems.tsx:23]: Deu erro não, mas vou alterar mesmo assim.

Comment (ericbraz) [reply components/MenuList/ListingItems/ListingItems.tsx:23]: No error found but I'll change it anyway.

**Review by brunorplima — CHANGES_REQUESTED** [review]
I just noticed something else that should be changed, other than that I think it's good to go.

There is one thing that can happen but should not. It's quite possible that they run out of a menu item during their work at the moment that a user is inside of that item and the user is about to add that item to the cart. If they're out of that menu item users shouldn't be allowed to add it to the cart, if they were already on the page of that menu item. Do that by disabling the add to cart button if the item is over.

**Review by brunorplima — CHANGES_REQUESTED** [review]
Nice work on preventing the user from adding the recently sold-out item.
I will have to apologize because I should've asked you this before...
I also noticed that in the case someone had already added that menu item to the cart but was not done choosing everything and during his browsing the item runs out, the user is still able to place an order on the `/checkout` page. There should also be some logic on that page that will open a dialog informing the user that a specific item he's added to the cart was recently sold out. This Dialog should only have an OK button to close the dialog.
Also add some logic that will automatically remove that item from the cart.

**Pull request merged**